<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wgt_Scope</class>
 <widget class="QWidget" name="wgt_Scope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wgtControls" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tbChannels">
         <attribute name="title">
          <string>Channels</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tblTraces">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="topMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="chkAverage">
                  <property name="text">
                   <string>Average FM</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spnAverage">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>50</number>
                  </property>
                  <property name="value">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="topMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string># Samples</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmbnumSamples">
                  <item>
                   <property name="text">
                    <string>250</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>500</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>1000</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2000</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>5000</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>10000</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>25000</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCustomPlot" name="plotScope" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
